### PR TITLE
[IMP] mail: restore warning notification for upload file

### PR DESCRIPTION
**Pull request by phenix-factory**

### CODE REVIEW

File: addons/mail/static/src/new/composer/composer.js [hunk 1]
```diff
@@ -340,6 +340,11 @@ export class Composer extends Component {
             this.attachmentUploader?.clear();
             this.props.composer.textInputContent = "";
             el.focus();
+        } else {
+            this.env.services.notification.add(
+                this.env._t("Please wait while the file is uploading."),
+                { type: "warning" }
```
Comment (tsm-odoo) [inline addons/mail/static/src/new/composer/composer.js:346]: The issue is that this message will also be displayed when pressing `Enter` on an empty composer. I think this should be an `else if` instead.

File: addons/mail/static/src/new/composer/composer.js [hunk 2]
```diff
@@ -340,6 +340,11 @@ export class Composer extends Component {
             this.attachmentUploader?.clear();
             this.props.composer.textInputContent = "";
             el.focus();
+        } else {
```
Comment (tsm-odoo) [inline addons/mail/static/src/new/composer/composer.js]: The issue is that this message will also be displayed when pressing `Enter` on an empty composer. I think this should be an `else if` instead.

Comment (phenix-factory) [reply addons/mail/static/src/new/composer/composer.js]: Yes, else is too generic for this message. I didn't think about the empty message.
Now it test that some attachment are still uploading

**Review by tsm-odoo — APPROVED** [review]

File: addons/mail/static/tests/new/discuss/discuss_tests.js [hunk 3]
```diff
@@ -1467,3 +1468,97 @@ QUnit.test(
...
+            {
+                body: "newest message",
+                model: "mail.channel",
+                res_id: mailChannelId2,
+            },
+        ]);
+        const { openDiscuss } = await start({
+            discuss: {
+                context: {
+                    active_id: `mail.channel_${mailChannelId2}`,
```
Comment (tsm-odoo) [inline addons/mail/static/tests/new/discuss/discuss_tests.js:1519]: @phenix-factory Uups, I didn't see this one, but I think we can do a pass to remove those at the end :)

Comment (phenix-factory) [reply addons/mail/static/tests/new/discuss/discuss_tests.js:1519]: Next PR will do it

**Pull request merged**